Patch in a placeholder to "download" the gradle distfile from the place
where MacPorts has already downloaded it.
--- gradle/wrapper/gradle-wrapper.properties.orig	2020-01-20 14:52:49.000000000 -0600
+++ gradle/wrapper/gradle-wrapper.properties	2021-09-21 22:17:50.000000000 -0500
@@ -2,4 +2,4 @@
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-5.6.2-bin.zip
+distributionUrl=file\://@DISTPATH@/gradle-5.6.2-bin.zip
